tools/setup-buildtools.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
 if [ -f /bin/yum ]; then
+if [ `cat /etc/os-release | grep CentOS-7` != "" ]; then
```

Comment:
nit: this one is a bit sketchy..

Reply:
I mean maybe we need to have a separate notes / document in markdown format that elaborates on this, build toos setup depending on OS, etc. Maybe in the Linux build instructions doc.

Reply:
I think it would be better to move all these installations to docker file, since docker image creation happens during the build